seismic_site/shared-templates/base.html

```diff
@@ -9,6 +9,7 @@
     <link crossorigin="anonymous" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Muli|Titillium+Web&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap" rel="stylesheet">
```

Comment:
General idea: I'd rather we didn't add any other fonts for now and use the ones we have in spite of the UX specs, as per the discussion in PR #48 
We're going to settle on a set of fonts sooner or later but until then I think we should keep them consistent